config/betamocks/services_config.yml

```diff
@@ -819,17 +819,6 @@
         :uid_location: 'query'
         :uid_locator: 'edipi'
 
-# Lighthouse Benefits Education API (GI Bill Statement of Benefits - GIBS)
-- :name: "Lighthouse Benefits Education"
-  :base_uri: <%= "#{URI(Settings.lighthouse.benefits_education.host).host}:#{URI(Settings.lighthouse.benefits_education.host).port}" %>
-  :endpoints:
-  - :method: :get
-    :path: '/services/benefits-education/v1/education/chapter33'
-    :file_path: "/lighthouse/benefits_education"
-    :cache_multiple_responses:
-      :uid_location: "query"
-      :uid_locator: "icn"
-
 # Lighthouse Benefits Claims API
 - :name: "Lighthouse Benefits Claims"
```

Comment:
I would rename this to Lighthouse and add a comment above the first entry that says `# Benefits Claims`